projboard/tests/test_client.py

```diff
@@ -143,3 +143,16 @@ def test_aboutpage(client):
     template_names = set(tmpl.origin.template_name for tmpl in response.templates)
     # And check if that 'about/about.html' is in the set.
     assert 'about/about.html' in template_names
+
+
+def test_login(client):
```

Comment:
Here you are just verifying login page returned

You should propose another test for authenticated login
Please refer to pytest-django docs

You should create a user with a fixture 
and login with its credentials

Reply:
@yaelnoyman1 
Please add test for login with Creating new User and use the password created,
Please refer this 2nd example
https://pytest-django.readthedocs.io/en/latest/helpers.html#id2

Reply:
done